extra/cockpit/foreman-cockpit-session

```diff
@@ -1,302 +1,377 @@
-#!/usr/bin/env ruby
+#!/usr/bin/tfm-ruby
```

Comment:
wont this break on deb? But possibly also EL-8 (unsure about that one).

Reply:
Right, we'll have to mangle it in packaging